website/docs/reference/commands/version.md

```diff
@@ -4,10 +4,20 @@ sidebar_label: "version"
 id: "version"
 ---
 
-The `--version` command-line flag returns information about the currently installed version of dbt Core or the dbt Cloud CLI.
+The `--version` command-line flag returns information about the currently installed version of dbt Core or the dbt Cloud CLI. This flag is not supported when invoking dbt in other dbt Cloud runtimes (for example, the IDE or scheduled runs).
 
-This flag is not supported when invoking dbt in other dbt Cloud runtimes (for example, the IDE or scheduled runs).
+- **dbt Core** &mdash; Returns the installed version of dbt-core and the adapters (which is one local bundle).
+- **dbt Cloud CLI** &mdash; Returns the installed version of the [dbt Cloud CLI](/docs/cloud/cloud-cli-installation) and, for the other `dbt_version` values, the _latest_ version of the dbt runtime in dbt Cloud.
 
+
+## Versioning
+To learn more about release versioning for dbt Core, refer to [How dbt Core uses semantic versioning](docs/dbt-versions/core#how-dbt-core-uses-semantic-versioning). 
+
+If the [Keep on latest version](/docs/dbt-versions/upgrade-dbt-version-in-cloud#keep-on-latest-version) setting is selected in dbt Cloud, then `dbt_version` will be the latest (continuous) release version. This also follows semantic versioning guidelines, in the format `YYYY.xx.yy`, with the major version representing the year (for example, `2024.04.1234`).
```

Comment:
sugg active voice
```suggestion
If dbt Cloud is set to [[Keep on latest version](/docs/dbt-versions/upgrade-dbt-version-in-cloud#keep-on-latest-version), then `dbt_version` uses the latest (continuous) release version. This follows semantic versioning, using `YYYY.xx.yy` format, where the year is the major version (for example, 2024.04.1234)
```

Reply:
i'm not sure if i understand where the `1234` comes from or what it means - if its semantic versioning, is it the patch? i thought the semantic version guidelines were more like 'MAJOR.MINOR.PATCH (e.g., 1.0.0)' format? idk I'm not a semantic version expert but the 1234 may confuse users?

Reply:
@mirnawong1 : "If dbt Cloud is set to..." is still passive voice (not active voice). i really like your sugg for conciseness, thanks!